pinot-core/src/main/java/org/apache/pinot/core/transport/ServerResponse.java

```diff
@@ -81,5 +86,7 @@ void receiveDataTable(DataTable dataTable, int responseSize, int deserialization
     _dataTable = dataTable;
     _responseSize = responseSize;
     _deserializationTimeMs = deserializationTimeMs;
+    _serverProcessingTimeMs =
```

Comment:
So the way this is set on the server, this is ServerQueryPhase.QUERY_PROCESSING_TIME which is not the TOTAL_QUERY_TIME so won't include the conversion of DataTable.toBytes(). To measure the true wall clock time on the server, we need totalQueryTime I believe